library/src/main/java/org/xmtp/android/library/ConversationV2.kt

```diff
@@ -30,6 +30,7 @@ data class ConversationV2(
     val context: Invitation.InvitationV1.Context,
     val peerAddress: String,
     val client: Client,
+    val createdAt: Long? = null,
```

Comment:
Same comments as iOS I think createdAt feels more like a date since this is taking NS lets make that more clear

```suggestion
    val createdAtNs: Long? = null,
```